### PR TITLE
Make header responsive

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

### CODE REVIEW

File: pmg/static/resources/css/style.scss [hunk 1]
```diff
@@ -213,6 +234,10 @@ article.page {
   img {
     margin-bottom: 10px;
   }
+
+  @media (max-width:768px) {
+    margin-top: 10px;
+  }
 }
 
 .correct-this-page-footer {
```
Comment (longhotsummer) [inline pmg/static/resources/css/style.scss]: We need an additional margin-bottom here of about 20px.

Comment (longhotsummer) [inline pmg/static/resources/css/style.scss]: Alternatively, let's just give .footer-1 some margin-top. The idea is to stop the content from being flush against it.

**Comment by longhotsummer** [comment]
This is looking great.

A small tweak which would be nice (but isn't critical) is to make form submission buttons (such as login, password reset, etc.) full width on mobile. It makes them clearer tap targets.

I generally add a help class such as this:

```css
.full-xs {
  @media (max-width: 767px) {
    width: 100%;
  }
}
```

**Comment by longhotsummer** [comment]
Btw, I'm happy to merge this and keep iterating. There are no show-stoppers here.

**Comment by xybrnet** [comment]
@longhotsummer I've made the changes.

**Pull request merged**